### DISCUSSION TITLE
When creating a hardlink to file, there should be a default filename filled in, equal to the filename of the file

### DISCUSSION THREAD

**Discussion by menkaur** [jarun/nnn#1929] [Ideas]
Currently, you can create a hardlink by selecting an item and pressing n-h, either in the same folder, or in a new folder.
nnn does not set default filename for the hardlink, so you have to enter it from memory.
I would like to change this behavior. nnn should propose default filename for the hardlink, either the filename itself, or something like {filename}\_hardlink\_{number}.{extention} if the filename already exists in the destination folder
This behavior would greatly simplify working with hardlinks, at least, in my workflow

**Comment by N-R-K** [comment]
> nnn does not set default filename for the hardlink

This is already the default behavior on `master` if you have exactly 1 item selected.

Which version of nnn are you using?

**Comment by menkaur** [comment]
I'm using 5.0

**Reply by N-R-K** [reply]
Okay, I see. The prefill behavior also requires that `-u` flag is set.
